debugger.md

```diff
@@ -1,60 +1,48 @@
 * This will create a launch.json file. Enter the name of the executable in 'program' field (including the path). This could be something like *"${workspaceRoot}/bin/Debug/dnxcore50/osx.10.11-x64/HelloWorld"*. 
 * For attach scenarios modify the processName attribute of launch.json
 * If you want 'compile' support for F5 you need to create a tasks.json file . There are multiple ways to do this.
   * You can get it [here](https://github.com/OmniSharp/omnisharp-vscode/blob/dev/template-tasks.json) and put it manually next to your launch.json file 
-  * Or you can open the command pallette in VSC (F1) and run the command *"Debugger: Add tasks.json"* (Make sure you have installed Mono to be able to do so, see above. If you don't have Mono installed you might get an error saying *Omnisharp not running*.)
-  * Or if you just want debugging, you can comment out 'preCompileTask' field​
-  * Modify your project.json to reference "NETStandard.Library":"1.0.0-rc3-*"
-   * Currently dotnet new creates a project.json that references a version that's too old. You should also get a hint to modify this.
+  * Or you can open the command palette in VSC (F1) and run the command *"Debugger: Add tasks.json"* (Make sure you have installed Mono to be able to do so, see above. If you don't have Mono installed you might get an error saying *Omnisharp not running*.)
+  * Or if you just want debugging, you can comment out 'preCompileTask' field​ in launch.json.
+* Modify your project.json to reference "NETStandard.Library":"1.0.0-rc3-*"
+   * Currently *dotnet new* creates a project.json that references a version that's too old. You should also get a hint to modify this.
    * Afterwards run *dotnet restore*.
 * In case you get a restore error due to lack of a NuGet.Config file, just create this file in the root directory of your project. You can find a sample [here](https://github.com/Microsoft/MIEngine/blob/abeebec39221c654bd69a0d2bcadca6a4a0d0392/tools/InstallToVSCode/CLRDependencies/NuGet.Config). 
 
 ####Debugging Code compiled on another computer
-* The target app binaries must be built with Portable PDBs for symbol loading to work. 
-* If the target binary is built on Linux / OSX, dotnet CLI will produce portable pdbs by default so no action is necessary. 
-* If the target binary is built on windows, the pdb needs to be converted: [How to convert Windows PDBs to Portable](https://microsoft.sharepoint.com/teams/DD_VSPlat/Diagnostics/_layouts/15/WopiFrame.aspx?sourcedoc={872c5298-6f17-4960-a5a0-acc4f215e730}&action=edit&wd=target%28%2F%2FMDD%2FXPlat%20CLR.one%7C1b640b59-6617-4452-b360-c24e9d5cad48%2FHow%20to%20convert%20Windows%20PDBs%20to%20Portable%7C49651c0c-f5a0-4d77-9478-9a233f0bf345%2F%29)
+* The target app binaries must be built with Portable PDBs for symbol loading to work. On Window Portable PDBs are only created when using dotnet cli for compilation.
```

Comment:
This isn't correct. You can get portable PDBs with msbuild, and they aren't generated by default with .NET CLI. See the email I sent you.